shinken/objects/config.py

```diff
@@ -1258,23 +1258,6 @@ def remove_exclusions(self):
     def clean(self):
         self.services.clean()
 
-    # In the scheduler we need to relink the commandCall with
-    # the real commands
-    def late_linkify(self):
```

Comment:
I don't get where this job is done now. I guess linkify still have to occur somehow right?